moveit_setup_assistant/templates/moveit_config_pkg_template/launch/trajectory_execution.launch.xml

```diff
@@ -7,9 +7,9 @@
   <param name="moveit_manage_controllers" value="$(arg moveit_manage_controllers)"/>
 
   <!-- When determining the expected duration of a trajectory, this multiplicative factor is applied to get the allowed duration of execution -->
-  <param name="trajectory_execution/allowed_execution_duration_scaling" value="1.2"/> <!-- default 1.2 -->
+  <param name="allowed_execution_duration_scaling" value="1.2"/> <!-- default 1.2 -->
   <!-- Allow more than the expected execution time before triggering a trajectory cancel (applied after scaling) -->
-  <param name="trajectory_execution/allowed_goal_duration_margin" value="0.5"/> <!-- default 0.5 -->
+  <param name="allowed_goal_duration_margin" value="0.5"/> <!-- default 0.5 -->
   
```

Comment:
Why did you reverted this? IMHO it makes sense to structure the param space using the "trajectory_execution" ns.
Furthermore, the namespace is needed for dynamic reconfigure. This was the point of #68.